src/Posts/Post.jsx

```diff
@@ -229,6 +229,20 @@ const addNewCommentFn = (newComment) => {
   State.update(state.comments.push(newComment));
 };
 
+const PromptSignUpWrapper = ({ children }) => {
+  const url = "/signup?";
+
+  if (context.accountId) {
+    return children;
+  }
+
+  return (
+    <a href={url} target="_blank" style={{ textDecoration: "none" }}>
```

Comment:
@jackson-harris-iii  I think we should always default to our custom `<Link>` component